efficientnet/scripts/convert_efficientnet.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
```

Comment:
This script throws the following error

```sh
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~ ............................. ~
~ Converting efficientnet-b0... ~
~ ............................. ~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/manrajsingh/github/tfjs-models/efficientnet/dist/efficientnet-source/scripts/load_efficientnet.py", line 156, in <module>
    import eval_ckpt_main
ModuleNotFoundError: No module named 'eval_ckpt_main'
```

Kindly look into the same

Reply:
Recent updates to `tensorflowjs_converter` have introduced a bug which I could not fix for hours. I have tried to roll back, but to no avail. The issue is explained [here](https://github.com/tensorflow/tfjs/issues/1726).